test/test_browser.py

```diff
@@ -3018,8 +3018,12 @@ def test_glfw_events(self):
     self.btest('test_glfw_events.c', args=['-sUSE_GLFW=2', "-DUSE_GLFW=2", '-lglfw', '-lGL'], expected='1')
     self.btest('test_glfw_events.c', args=['-sUSE_GLFW=3', "-DUSE_GLFW=3", '-lglfw', '-lGL'], expected='1')
 
-  @no_wasm64('SDL2 + wasm64')
   @requires_graphics_hardware
+  def test_glfw3_hi_dpi_aware(self):
+    self.btest_exit('test_glfw3_hi_dpi_aware.c', args=['-sUSE_GLFW=3', '-sASYNCIFY', '-lglfw', '-lGL'])
```

Comment:
Does this test use `-sASYNCIFY` ?  Also I don't think you need `-lglfw` if you use  `-sUSE_GLFW=3`.

Reply:
Yes you are correct about `-sASYNCIFY`, that was a leftover of a previous development where I was calling sleep so that I could get a chance to look at the messages/console.

`-lglfw` is also not needed but I copy/pasted it from another glfw test.

I have removed both